index.js

```diff
@@ -100,6 +100,21 @@ class PQueue {
 		return this._pendingCount < this._concurrency;
 	}
 
+	get concurrency() {
+		return this._concurrency;
+	}
+
+	set concurrency(newConcurrency) {
```

Comment:
For API consistency (e.g. pause), `updateConcurrency` or `setConcurrency` as a method may be better. It would also more naturally provide a `Promise` response.

Reply:
I prefer a getter/setter property whenever possible. It let’s you also read the value.

Can you elaborate on what you mean by “promise response”?

Reply:
Explanation here: https://github.com/sindresorhus/p-queue/pull/52#discussion_r244627461

Was concerned with synchronous blocking in a setter.